### PR TITLE
Added List Certificate Function

### PR DESCRIPTION
**Pull request by psgoundar**
This is a list function that show all the issues Certificates from the /etc/openvpn/easy-rsa/pki/index.txt file and formats it into a table to show the Status, Name and Expiration Dates.

### CODE REVIEW

**Review by randshell — COMMENTED** [review]
Fixes #563

**Comment by randshell** [comment]
At the moment `index.txt` contains revoked users too so this function will show them #566

**Comment by psgoundar** [comment]
For my personal Use case I need to list revoked users as well to see them. But I can see why that would not be needed. I can take remove all revoked users or potentially in a future update add another submenu to  list All or only valid Certificates?

**Comment by randshell** [comment]
@psgoundar I'd like to see revoked users too but there are some user who asked to not show them https://github.com/angristan/openvpn-install/issues/566#issuecomment-606561819.

My previous comment was to link to the issue not to request changes, @angristan will decide on this.

**Comment by randomshell** [comment]
Fixes #563

File: openvpn-install.sh [hunk 1]
```diff
@@ -1248,10 +1281,11 @@ function manageMenu () {
 	echo "What do you want to do?"
 	echo "   1) Add a new user"
 	echo "   2) Revoke existing user"
-	echo "   3) Remove OpenVPN"
-	echo "   4) Exit"
-	until [[ "$MENU_OPTION" =~ ^[1-4]$ ]]; do
-		read -rp "Select an option [1-4]: " MENU_OPTION
+	echo "   3) List Current Issued Certificates"
+	echo "   8) Remove OpenVPN"
+	echo "   9) Exit"
```
Comment (angristan) [inline openvpn-install.sh]: 3, 8, 9?

**Review by angristan — CHANGES_REQUESTED** [review]

File: openvpn-install.sh (hunk 1 above)
Comment (psgoundar) [reply openvpn-install.sh]: I've got multiple other options that I've added to my repo And I was planning on slowly submitting all the changes up to the main repo.
And I didn't want to continually be renumbering the remove and Exit options
Such as:
        echo "   1) Add a new user"
	echo "   2) Revoke existing user"
	echo "   3) List Current Issued Certificates"
	echo "   4) List Current Active Users"
	echo "   5) Backup Configuration"
	echo "   6) Restore Configuration from Backup *Incomplete"
	echo "   7) Sync Configuration to Alternate Servers *Incomplete"
	echo "   8) Remove OpenVPN"
	echo "   9) Exit"

Comment (angristan) [reply openvpn-install.sh]: Please do the appropriate changes for this PR :)

Comment (psgoundar) [reply openvpn-install.sh]: I think I totally Screwed up my branch. I will close this pull Request. And pull another with the neccessary Changes.

**Comment by psgoundar** [comment]
Repulled Main Repo and  Redoing changes

File: openvpn-install.sh (hunk 1 above)
Comment (angristan) [reply openvpn-install.sh]: Ok :)

**Pull request closed without merging**